.wordlist.txt

```diff
@@ -87,6 +88,8 @@ ENDPGM
 EPEL
 EPYC
 ESXi
+EU
+ExLlama
```

Comment:
A standalone Python/C++/CUDA implementation of Llama for use with 4-bit GPTQ weights, designed to be fast and memory-efficient on modern GPUs.